### DISCUSSION TITLE
GLM-130b Model Weights

### DISCUSSION THREAD

**Discussion by conceptofmind** [THUDM/GLM-130B#15] [Q&A] [answered]
Hi @Sengxian ,

I was wondering what the rules were for sharing the model weights of GLM-130b directly with members of a research team working on the same project. If we have 3 or 4 people running evaluation on benchmarks such as KLUE, can the weights be shared if contained to that single project instance? Everything is non-commercial. **Currently no weights have been shared and only a single person with access is running evaluation**. 

Thank you again for all of your help!

Enrico

**Comment by Sengxian** [comment, marked as answer]
As an official response, we still suggest your team members to sign the license. But as long as your use is for non-commercial research purposes and does not cause legal and ethical concerns, we think it won't be a problem.

**Reply by conceptofmind** [reply]
Hi @Sengxian ,

I appreciate the immediate response.

I will ensure all participating members sign the license before sharing. It is all non-commercial research.

Thank you!

Enrico
